public/app/features/alerting/unified/components/receivers/ReceiversAndTemplatesView.tsx

```diff
@@ -18,26 +21,54 @@ interface Props {
 }
 
 export const ReceiversAndTemplatesView = ({ config, alertManagerName }: Props) => {
-  const isCloud = alertManagerName !== GRAFANA_RULES_SOURCE_NAME;
+  const isGrafanaManagedAlertmanager = alertManagerName === GRAFANA_RULES_SOURCE_NAME;
```

Comment:
this comparison is made in a lot of places (unrelated to this PR), maybe we should move this to a common method that can be reused? Right now it's a pretty simple check though, so not sure it's worth it.

Reply:
I think this is simple enough for now, but yea it could be more functional :)